newsControllers/newsControllers

```diff
@@ -0,0 +1,15 @@
+const {fetchTopics } = require('../newsModels/newsModels')
+
+
+exports.getTopics = (req, res) => {
```

Comment:
next is missing from your parameters so will be undefined in your code

Reply:
Yes, I got a 'next is undefined' message at some point - but it resolved itself after I made some other changes, so I did not know the reason. Thank you for highlighting.